<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>258</width>
      <height>147</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btnLogin">
           <property name="text">
            <string>登录</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRegister">
           <property name="text">
            <string>注册</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLogout">
           <property name="text">
            <string>退出登录</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btnSendMessage">
           <property name="text">
            <string>发消息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnJoinGroup">
           <property name="text">
            <string>加入群</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSendGroupMessage">
           <property name="text">
            <string>群发消息</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>服务器：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leServerIP"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnConnect">
         <property name="text">
          <string>连接</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="teMsgRecv">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>167</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
